### DISCUSSION TITLE
无法加载任何漫画

### DISCUSSION THREAD

**Discussion by Fanshi233** [xiaoyaocz/flutter_dmzj#148] [General]
![Screenshot_20230327-004736](https://user-images.githubusercontent.com/93038980/227791119-7f76d226-935f-43fa-a410-0fcb465f79c2.jpg)
任何漫画点进去都是这样

**Comment by xiaoyaocz** [comment]
需要登录以及部分漫画有等级限制

**Comment by KT-71** [comment]
![Screenshot_20230328-094439](https://user-images.githubusercontent.com/16741389/228107275-d37159c3-fc62-429d-baa0-a40adad8b5e8.png)
發生同樣問題
確認有登入且m.idmzj能夠觀看同樣的漫畫
版本v2.0.5

或者其實網頁版不受等級限制？
